server/views/components/promo/index.njk

```diff
@@ -9,7 +9,9 @@
 <a href="{{ promo.article.url }}" class="promo {% for modifier in promo.modifiers %}promo--{{ modifier }} {% endfor %}">
   <div class="promo__image-container">
 
-    {% componentV2 'image', promo.article.thumbnail, {}, {lazyload: true, sizesQueries: sizes} %}
+    {% if promo.article.thumbnail.contentUrl %}
```

Comment:
Check for the presence of a thumbnail (given they might not exists in older posts). An alternative could be to add a standard placeholder thumbnail to all posts that don't have a `post_thumbnail` key in their json response